src/to-have-style.js

```diff
@@ -17,9 +17,12 @@ function getStyleDeclaration(document, css) {
 }
 
 function isSubset(styles, computedStyle) {
-  return Object.entries(styles).every(
-    ([prop, value]) =>
-      computedStyle.getPropertyValue(prop.toLowerCase()) === value,
+  return (
+    !!Object.keys(styles).length &&
```

Comment:
I guess you added this because we now need to return `false` if the object is empty, right?

Also, why not the more explicit alternative:

```suggestion
    Object.keys(styles).length > 0 &&
```

Reply:
> I guess you added this because we now need to return `false` if the object is empty, right?

Yes.